### DISCUSSION TITLE
new node unlocked links

### DISCUSSION THREAD

**Discussion by 970bih30-boy** [nfriedly/node-unblocker#193] [General]
just want a node unblocker link

**Comment by 970bih30-boy** [comment]
or an alt link to it

**Comment by ghost** [comment]
proxy.crove.io iirc, but this is exactly the wrong place to ask.

**Comment by 970bih30-boy** [comment]
huh

**Comment by 970bih30-boy** [comment]
do you know where to ask

**Comment by nfriedly** [comment]
Yeah, @MadKamel is correct, github issues isn't really an appropriate place for this kind of discussion.  https://discord.gg/unblock is one place you could try.

~~I'm going to go ahead and close this, though.~~

Update: I went ahead and turned on github discussions and converted this issue to a discussion
